### ISSUE TITLE
Submit draft comments to developers through the review page

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-server#12660]
User story:

> As a reviewer, I want to use the Code Manger to enter comments then submit those comments to the developer through the reviewer tools.

We have API calls to support adding comments (and some UI [has landed](https://github.com/mozilla/addons-code-manager/issues/112)). Next, the reviewer page needs to expose the comments and let the reviewer submit them. 

In our original [meeting](https://docs.google.com/document/d/1y4crrHxhGejb3eZ6HpjtBdDqONJakfwtfjaOqwq-qZo/edit), we had this plan:

> The file and diff viewer should provide a button to complete the review. If the reviewer clicks on the button, the review page will load, showing the generated (non-editable) review text based on the annotations made in the file/diff viewer above the “Review Actions” section.
>
> The review text should group the issues by canned response identifier, so that multiple occurrences of the same issue don’t lead to the issue itself being repeated. The already existing comments box will become optional, so that a reviewer can add further messaging if necessary.
>
> Each review entry should include files and lines where they occurred. They should become links to the file viewer, on the email sent to the developer.

**Comment by EnTeQuAk** [comment]
> Each review entry should include files and lines where they occurred. They should become links to the file viewer, on the email sent to the developer.

Currently the reviewer tools are only sending plain text emails so this won't work necessarily. Unsure if we want to break that pattern.

In any case, currently developers don't have access to the reviewer tools / code manager - so let's split this into a separate issue maybe?

**Comment by kumar303** [comment]
> Currently the reviewer tools are only sending plain text emails so this won't work necessarily. Unsure if we want to break that pattern.
> 
> In any case, currently developers don't have access to the reviewer tools / code manager - so let's split this into a separate issue maybe?

Yeah, let's start with plain text. I updated the description of this issue.

Also, just FYI, I have proposed a quick fix UI patch that will at least let reviewers copy / paste the text from Code Manager into the reviewer tools. You can see screenshots here: https://github.com/mozilla/addons-code-manager/pull/1170

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-21
